lib/smoke_test.rb

```diff
@@ -0,0 +1,61 @@
+require 'net/http'
+
+class SmokeTest
+  def initialize(urls)
```

Comment:
Instead of passing the urls as variables, which can be messy sometimes, you could get those from the secrets file. So you could instruct your users, through the readme, to add an extra field for the urls and then fetch it here

Reply:
Thanks @Levia I will change this!